docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -576,6 +584,59 @@ the two columns or sets of columns must have the same characteristics (data type
 the foreign key in _table_ is the column being defined; with the foreign key clause, the foreign key is the column or set of columns specified in
 the foreign key clause. for information about _ref-spec_, see references _ref-spec_ not enforced.
 
+<<<
+* ENABLE ALL [UNIQUE] INDEXES
+
+** ENABLE ALL UNIQUE INDEXES
++
+enables all unique indexes on the table.
+
+** ENABLE ALL INDEXES
++
+enables all indexes (including normal indexes and unique indexes) on the table.
+
+* _index-name_

```

Comment:
Again, I would not make a distinction at this level between unique and non-unique indexes. Suggested wordsmith: "* _index-name_   <blank lines> is an SQL identifier that specifies the simple name for the index. You cannot qualify index-name... <the rest of your text is fine>"

Reply:
Thanks @DaveBirdsall, I've udpated :)